Spigot-Server-Patches/0179-Allow-Reloading-of-Command-Aliases.patch

```diff
@@ -0,0 +1,29 @@
+From 094172b70ec10ecd48d1093f0425f88afcce397e Mon Sep 17 00:00:00 2001
+From: willies952002 <admin@domnian.com>
+Date: Mon, 21 Nov 2016 23:11:25 -0500
+Subject: [PATCH] Allow Reloading of Command Aliases
+
+Reload the aliases stored in commands.yml
+
+diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+index d5e602f..f429562 100644
+--- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
++++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
+@@ -1861,5 +1861,14 @@ public final class CraftServer implements Server {
+         }
+         return entity.getBukkitEntity();
+     }
++
++    @Override
++    public void reloadCommandAliases() {
++        commandsConfiguration = YamlConfiguration.loadConfiguration(getCommandsConfigFile());
```

Comment:
This section needs work.

You need to remove the commands BEFORE reloading the configuration file, as if you remove a command from commands.yml, the way you have this, will not remove the command.